wfl/cli/commands/select.py

```diff
@@ -0,0 +1,50 @@
+from pathlib import Path
+import click
+from wfl.cli import cli_options as opt
+from wfl.configset import ConfigSet, OutputSpec
+import wfl.descriptors.quippy
+import wfl.select.by_descriptor
+from .descriptor import calculate_descriptor
+from wfl.select.simple import by_bool_func
+
+@click.command("cur")
+@click.option("--n-configs", "-N", type=click.INT, required=True,
+              help="number of configs to select")
+@click.option("--keep_descriptor", is_flag=True, help="keep the descriptor value in the final config file")
+@click.option("--kernel_exponent", type=click.FLOAT, help="exponent of dot-product for kernel")
+@click.option("--deterministic", is_flag=True, help="use deterministic (not stochastic) CUR selection")
```

Comment:
Do we want to add support for setting a seed for the stochastic one?

Reply:
added.